tests/models/detr/test_feature_extraction_detr.py

```diff
@@ -331,7 +341,7 @@ def test_call_pytorch_with_coco_panoptic_annotations(self):
         expected_class_labels = torch.tensor([17, 17, 63, 75, 75, 93])
         self.assertTrue(torch.allclose(encoding["labels"][0]["class_labels"], expected_class_labels))
         # verify masks
-        expected_masks_sum = 822338
+        expected_masks_sum = 822873
```

Comment:
The values for DETR, Conditional DETR, Deformable DETR and YOLOS all changed for the same test here. There are 535 pixels different across the 6, 800 * 1066 pixel masks, representing a 0.01% change. 

This is due to the [resizing of the annotation masks](https://github.com/amyeroberts/transformers/blob/4979808c16bc9b4afaf5ccef644a88b1a6de1bb7/src/transformers/models/detr/image_processing_detr.py#L555) now being performed by functionality in the [image transforms library](https://github.com/huggingface/transformers/blob/ab9fe45236cd99b8797df78219438f8f6662bb42/src/transformers/image_transforms.py#L267) (using Pillow), whereas it was [previously done by torch](https://github.com/huggingface/transformers/blob/ab9fe45236cd99b8797df78219438f8f6662bb42/src/transformers/models/detr/feature_extraction_detr.py#L477). This was done to make the preprocessing framework agnostic. 

Note: the `"nearest"` mode for torch interpolation is the same function as Open CVs. The equivalent for scipy/Pillow is `"nearest-exact"` c.f. [torch.nn.functional.interpolation documentation](https://pytorch.org/docs/stable/generated/torch.nn.functional.interpolate.html). When this is used, the number of pixels different is 2.